_guides/getting_started.md

```diff
@@ -57,64 +86,140 @@ best constant = 0.500000
 best constant's loss = 0.250000
 total feature number = 15
 ```
-## VW's diagnostic information
 
-There is a burble of diagnostic information which you can turn off with `--quiet`. However, it's worthwhile to first understand it, so let's work through it bit by bit.
+## Vowpal Wabbit diagnostics
+
+This section provides information on the various types of diagnostic information VW presents.
+
+**Use** `--quiet`  command to turn off diagnostic information in Vowpal Wabbit.
+
+### Hash function bits
+
+The following VW output shows the number of bits from the hash function:
 
 ```
 Num weight bits = 18
 ```
-Only 18 bits of the hash function will be used. That's much more than necessary for this example. You could adjust the number of bits using `-b bits`
+
+This diagnostic shows that the number of bits from the hash function is 18 (more than enough for this example). Use `-b bits` to adjust the number of bits for a hash function.
```

Comment:
@kumpera comments:
We use the bits that comes out of the hash function.

Changed